fix-point/ChangeLog.md

```diff
@@ -0,0 +1,3 @@
+# Changelog for fix-point
+
+## Unreleased changes
```

Comment:
I think you can remove changelog file as well because probably nobody will maintain it.